samples/modelbuilder/BinaryClassification_Sentiment_Razor/SentimentRazorML.ConsoleApp/ModelBuilder.cs

```diff
@@ -56,25 +56,15 @@ private static async Task DownloadData()
         {
             using (var client = new HttpClient())
             {
-                var response = await client.GetStreamAsync("https://archive.ics.uci.edu/ml/machine-learning-databases/00331/sentiment%20labelled%20sentences.zip");
+                var response = await client.GetStreamAsync("https://raw.githubusercontent.com/dotnet/machinelearning/master/test/data/wikipedia-detox-250-line-data.tsv");
```

Comment:
I would recommend a larger sample of the dataset. It's rather unlikely the model will be very good when trained on 250 rows. 

Background:
The 250 row sample was created for a unit test to check for equality without respect to how good a model it produces. Just as long as it produced the same model as last time, the unit test shows nothing has changed.

Reply:
Agreed this dataset is limiting. Unfortunately given the constraints around building a plug-and-play experience, re-hosting datasets, no feature engineering in Model Builder at the moment and the 160k rows dataset being imbalanced (90% non-toxic, 10% toxic), this is the "best" dataset for us to work with at the moment considering it's already part of the [getting started experience](https://dotnet.microsoft.com/learn/ml-dotnet/get-started-tutorial/intro) on the .NET site . However, if there is another version of the version of the dataset that can be consumed as-is with no manipulation from the user, I can switch the dataset then.